vignettes/using.Rmd

```diff
@@ -501,6 +501,8 @@ To specify that a page should be accessed via a navigation bar menu you use the
 <div id="pagenavmenu"></div>
 <script type="text/javascript">loadSnippet('pagenavmenu')</script>
 
+You can optionally provide icons for the navigation bar menus by including the `data-navmenu-icon` attribute when you first assign a page to a navigation bar menu.
```

Comment:
Can we point to somewhere in the docs for explaining what icons/syntax is supported?

Reply:
Sure! It's just a couple sections below in "Page Icons". I added a link to make it easier to find in 1c89f03afe816e6176962eb112af2d19f4805b47